config/locales/en.yml

```diff
@@ -1133,10 +1133,17 @@ en:
     logged_out: "Successfully logged out."
   notes_controller:
     research_notes: "Research notes"
-    thank_you_for_contribution: "Success! Thank you for contributing open research,
-      and thanks for your patience while your post is approved by <a href='/wiki/moderation'>community
-      moderators</a> and we'll email you when it is published. In the meantime, if
-      you have more to contribute, feel free to do so."
+    thank_you_for_contribution: |
+      <h2>Success! You did it! Hooray!</h2>
+      <hr>
+      <p>Thank you so much for contributing open research and being part of our community.</p>
```

Comment:
```suggestion
      <p>Thank you so much for contributing to open research and being a part of our community.</p>
```

Reply:
Hi @priyda, do you need any help with this? We're here to help!

Reply:
Hi, I have commited suggestion. Let me know if I need to make any more changes.